kiva/ui
# Storybook

To test:
```
git fetch
git checkout VUE-172_storybook
npm install
npm run storybook
```
This will spin up a local server with live reload that watches anything in the .storybook/stories directory and any components that those stories reference. So play around with a .stories.js file or say KvRadio.vue and you should see the live reload.

Nothing in storybook will get bundled with our regular Vue app, these are all dev-dependencies, things pretty self-contained in the .storybook directory.

You can export a static site by running 
```npm run build-storybook```
which we could host somewhere, or not. Let's chat more after standup tomorrow, I'll walk through a demo.
